atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/DbKvsPostgresGetCandidateCellsForSweepingTest.java

```diff
@@ -16,113 +16,18 @@
 
 package com.palantir.atlasdb.keyvalue.dbkvs.impl.postgres;
 
-import static org.junit.Assert.assertEquals;
-
-import java.util.List;
-
-import org.junit.Test;
-
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Iterators;
-import com.palantir.atlasdb.encoding.PtBytes;
-import com.palantir.atlasdb.keyvalue.api.CandidateCellForSweeping;
-import com.palantir.atlasdb.keyvalue.api.CandidateCellForSweepingRequest;
-import com.palantir.atlasdb.keyvalue.api.ImmutableCandidateCellForSweeping;
 import com.palantir.atlasdb.keyvalue.api.KeyValueService;
 import com.palantir.atlasdb.keyvalue.dbkvs.DbKeyValueServiceConfig;
 import com.palantir.atlasdb.keyvalue.dbkvs.DbkvsPostgresTestSuite;
 import com.palantir.atlasdb.keyvalue.dbkvs.impl.ConnectionManagerAwareDbKvs;
-import com.palantir.atlasdb.keyvalue.dbkvs.impl.SqlConnectionSupplier;
-import com.palantir.atlasdb.keyvalue.dbkvs.impl.sweep.DbKvsGetCandidateCellsForSweeping;
 import com.palantir.atlasdb.keyvalue.impl.AbstractGetCandidateCellsForSweepingTest;
-import com.palantir.common.base.ClosableIterator;
 
 public class DbKvsPostgresGetCandidateCellsForSweepingTest extends AbstractGetCandidateCellsForSweepingTest {
 
-    private static DbKeyValueServiceConfig config;
-    private static SqlConnectionSupplier connectionSupplier;
-
     @Override
     protected KeyValueService createKeyValueService() {
-        config = DbkvsPostgresTestSuite.getKvsConfig();
-        ConnectionManagerAwareDbKvs kvs = ConnectionManagerAwareDbKvs.create(config);
-        connectionSupplier = kvs.getSqlConnectionSupplier();
-        return kvs;
-    }
-
-    @Test
-    public void singleCellSpanningSeveralPages() {
-        new TestDataBuilder()
-                .put(10, 1, 1000)
-                .put(10, 1, 1001)
-                .put(10, 1, 1002)
-                .put(10, 1, 1003)
-                .put(10, 1, 1004)
-                .store();
-        List<CandidateCellForSweeping> cells = getWithOverriddenLimit(
-                conservativeRequest(PtBytes.EMPTY_BYTE_ARRAY, 2000L, Long.MIN_VALUE), 2);
-        assertEquals(ImmutableList.of(ImmutableCandidateCellForSweeping.builder()
-                .cell(cell(10, 1))
-                .isLatestValueEmpty(false)
-                .numCellsTsPairsExamined(5)
-                .sortedTimestamps(1000L, 1001L, 1002L, 1003L, 1004L)
-                .build()), cells);
-    }
-
-    @Test
-    public void returnFirstAndLastCellOfThePage() {
```

Comment:
This test still seems valuable, why did we get rid of this?

Reply:
This tested implementation-specific behavior that doesn't exist anymore